synthesizer/event.cpp

```diff
@@ -135,6 +136,35 @@ Event::~Event()
       }
 
 //---------------------------------------------------------
+//   NPlayEvent::NPlayEvent (beatType2metronomeEvent)
+//---------------------------------------------------------
+
```

Comment:
@lasconic All done. I ended up putting this in a new constructor for NPlayEvent. That seems to be the obvious place for it - I probably should have thought of it sooner.